src/discord.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const discordjs = require('discord.js')
+const getRepo = require('./parser').getRepoName
+
+var client = new discordjs.Client()
+var token = process.env[`npm_package_config_token`]
+
+var loggedIn = false;
+if (token) client.login(token).then(loggedIn = true).catch(console.error);
```

Comment:
Is it possible for the line after this to run before the `then`, or am I forgetting how the event loop works?

Reply:
The line after this will, without exception, execute before any `then` handlers. But also, `loggedIn = true` will execute before you actually login, because it isn't wrapped in a function

Reply:
I've reworked this; shouldn't be a problem now